app/src/Header.tsx

```diff
@@ -32,10 +33,10 @@ export const Header = () => {
           <IconSVG />
         </div>
         <div className="title">
-          <h3>Polkadot Cloud Docs</h3>
-          <p>
-            v0.2.14 <span className="beta">Beta</span>
-          </p>
+          <h3>
+            Polkadot Cloud<span className="beta">Beta</span>
+          </h3>
+          <p>v{pkg?.version}</p>
```

Comment:
The label next to the title doesn't work well on mobile unfortunately. I think just have it inline with `pkg?.version` is okay. We could even ditch the label altogether and make the version `0.2.15-beta`.

```suggestion
            Polkadot Cloud Docs
          </h3>
          <p>v{pkg?.version} <span className="beta">Beta</span></p>
```

Reply:
🤔 already checked on mobile and it looks ok to me. I wanted it to be in alignment with the footer but im ok to change it